src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -45,4 +90,47 @@ public void setup () {
         entityGenerator = new EntityGenerator(injector.getInstance(OctaneProvider.class));
     }
 
+    public Long createWorkSpace() {
+
+        String postUrl = connectionSettings.getConnectionSettings().getBaseUrl() + "/api/shared_spaces/" +
+                connectionSettings.getConnectionSettings().getSharedSpaceId() + "/workspaces";
+
+        String urlDomain = connectionSettings.getConnectionSettings().getBaseUrl();
+
+        JSONObject dataSet = new JSONObject();
+        JSONObject credentialsJson = new JSONObject();
+        credentialsJson.put("name", "test_workspace1");
+        credentialsJson.put("description", "Created from intellij");
+        JSONArray jsonArray = new JSONArray();
+        jsonArray.put(credentialsJson);
+        dataSet.put("data", jsonArray);
+
+        System.out.println(dataSet.toString());
+
+        OctaneHttpRequest postNewWorkspaceRequest = new OctaneHttpRequest.PostOctaneHttpRequest(postUrl, OctaneHttpRequest.JSON_CONTENT_TYPE, dataSet.toString());
+        OctaneHttpClient octaneHttpClient = new GoogleHttpClient(urlDomain);
+        octaneHttpClient.authenticate(new SimpleUserAuthentication(connectionSettings.getConnectionSettings().getUserName(), connectionSettings.getConnectionSettings().getPassword(), ClientType.HPE_MQM_UI.name()));
+        OctaneHttpResponse response = null;
+
+        try {
+            response = octaneHttpClient.execute(postNewWorkspaceRequest);
+
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
This is not handling the exception, the code would continue and throw a fatal null pointer exception later.
This won't help the one debugging it. We should use the logging framework to log an error message that is apropriate. Also from now on all Sys outs should be replaced by the logging framework.

In this case, after the exceptions is caught, junit should stop the execution of the tests in the subclass. since the req workspace creation has failed.